kolibri/core/assets/src/views/CoreInfoIcon/index.vue

```diff
@@ -58,6 +58,13 @@
     cursor: pointer;
   }
 
+  // Overwrite KICon styles to make it compatible with UIIcon
```

Comment:
This is a question out of scope of the review, just as a matter of interest - what's the reason for wrapping _KIcon_ in _UIIcon_?

Reply:
I believe that `UiIcon` gives the the `KIcon` a button-like appearance and functionality. We use it in different places, like the `<-` (back) button on some AppBars

Reply:
cc @nucleogenesis thoughts?